spec/hosts/roles/rke_spec.rb

```diff
@@ -34,19 +35,69 @@
         }
       end
 
-      lsst_sites.each do |site|
-        fqdn = "#{role}.#{site}.lsst.org"
-        override_facts(facts, fqdn: fqdn, networking: { fqdn => fqdn })
+      describe 'rke.tu.lsst.org', :sitepp do
```

Comment:
this is weird...

Reply:
The `        override_facts(facts, fqdn: fqdn, networking: { fqdn => fqdn })` is still going to be needed for each site.

Reply:
`networking: { fqdn => fqdn })` is still needed.